examples/README.md

```diff
@@ -166,9 +181,54 @@ example.
 
    </details>
 
+10. [**Exported memory**][exported-memory], explains how to read from 
+    and write to exported memory.
+   
+   _Keywords_: export, memory.
+
+   <details>
+   <summary><em>Execute the example</em></summary>
+
+   ```shell
+   $ cargo run --example exported-memory --release --features "cranelift"
+   ```
+
+   </details>
+
+### Imports
+
+11. [**Imported global**][imported-global], explains how to work with
+   imported globals: create globals, import them, get/set their value.
+   
+   _Keywords_: import, global.
+
+   <details>
+   <summary><em>Execute the example</em></summary>
+
+   ```shell
+   $ cargo run --example imported-globals --release --features "cranelift"
+   ```
+
+   </details>
+
+12. [**Imported function**][imported-function], explains how to define 
```

Comment:
I think GitHub markdown will number correctly for us if we just use `1.` everywhere; we can do that in a separate PR though, just putting the idea out there in case I forget

Reply:
Didn't know that. I'm trying:

```md
1. Hello
1. World

ozkedokze

3. Oh
1. Does it
1. Work?
```

1. Hello
1. World

ozkedokze

3. Oh
1. Does it
1. Work?